ReactWindows/ReactNative/Views/Image/ReactImageManager.cs

```diff
@@ -163,6 +165,29 @@ public void SetBorderRadius(Border view, double radius)
         }
 
         /// <summary>
+        /// Set the tint color of the image view.
+        /// </summary>
+        /// <param name="view">The image view instance.</param>
+        /// <param name="color">The tint color value.</param>
+        [ReactProp("tintColor")]
+        public void SetTintColor(Border view, uint? color)
+        {
+            var data = GetImageData(view);
+            if (data.TintColor != color)
+            {
+                data.TintColor = color;
+                if (EffectManager.IsCompositionSupported)
+                {
+                    data.EffectManager.SetTintColor(color.Value);
+                }
+                else
+                {
+                    Debug.WriteLine("Tint color isn't supported.");
```

Comment:
>Debug [](start = 20, length = 5)

We don't have to do this in this pass, but we may want to implement something in the Image.windows.js file to "console.warn" if EffectManager.IsCompositionSupported is false. We could do this by adding a constant to the ReactImageManager, and then checking that constant if either the `blurRadius` or `tintColor` prop is set. That way, the developer will get a "yellow box" warning if they set blurRadius / tintColor prop on a device that doesn't support it, instead of just a Debug.WriteLine that is likely to be ignored.

Reply:
Yeah, totally agree.